components/Cards/FriendCard.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import Avatar from "@material-ui/core/Avatar";
+import AllOutIcon from "@material-ui/icons/AllOut";
+import DoneIcon from "@material-ui/icons/Done";
+import EcoIcon from "@material-ui/icons/Eco";
+import LockIcon from "@material-ui/icons/Lock";
+
+const useStyles = makeStyles({
+    box: {
+      borderRadius: "0px 15px 15px 0px",
+      // HOW TO PUT MATERIAL UI COLORS HERE
+      background: "#977BBF",
+    },
+  }),
+  FriendCard = () => {
+    const classes = useStyles();
+    return (
+      <Box className={classes.box}>
+        <table class="mx-auto max-w-4xl w-full whitespace-nowrap rounded-lg divide-y overflow-hidden">
+          <tbody>
+            <tr>
+              <td class="px-6 py-4">
```

Comment:
interesting use of `table`. why did you choose to use it? I think `table` should be reserved for semantic meaning, like an actual "table". I think having a div with flex should do just the trick :)

Reply:
I would agree. Tables should be reserved for transferring data sets due to their connotation. We don't have a huge emphasis on accessibility right now but taking that into consideration, using tables wouldn't be very appropriate.

Reply:
![image](https://user-images.githubusercontent.com/39652186/121115439-13493300-c7e3-11eb-8ea1-a84136807900.png)
changed back to list item instead